docs/user_guide/configuring.rst

```diff
@@ -48,6 +48,17 @@ an external site. You can add external links to the nav bar like so:
      ]
    }
 
+Configure the sidebar
+=====================
+
+``pydata_sphinx_theme`` provides two new sidebar items by default:
+
+- ``nav-bs.html`` - a bootstrap-friendly navigation section
```

Comment:
Maybe can use some more clear name like `sidebar-nav` ? (Because "nav" could also be about the navbar)

Reply:
The "bs" is for bootstrap I suppose? (in my head it's "boots-trap", but I realize now that is wrong :-))

Reply:
yep, I figured the `bs-` makes sense just to make it clear that it's a theme-specific thing. I guess we could also use `pydata-sidebar-nav` or something? Though if you're fine with `bs-` I'd probably just stick with that...

Reply:
what would you like me to do here? do you prefer a re-name?

Reply:
I updated this to `sidebar-nav-bs` in case that's what you wanted, LMK if you prefer something different

Reply:
Yep, let's not be too picky about the names;)